ERCS/erc-7291.md

```diff
@@ -0,0 +1,696 @@
+---
+eip: 7291
+title: Purpose bound money
+description: An interface extending ERC-1155 to implement purpose-bound money, a hybrid form of programmable payment and money.
+author: Orchid-Dev (@proj-orchid-straitsx), Victor Liew (@alcedo), Wong Tse Jian (@wongtsejian), Chin Sin Ong (@chinsinong)
+discussions-to: https://ethereum-magicians.org/t/eip-7291-purpose-bound-money/14973
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-24
+requires: 165, 173, 1155
+---
+
+## Abstract
+
+This proposal outlines a smart contract interface that builds upon the [ERC-1155](./eip-1155.md) standard to implement the purpose bound money (PBM) concept:
+
+- PBMs is comprised of a PBM wrapper and a digital money token that it wraps. A digital money token (e.g. stablecoins, central bank digital currency, tokenised bank deposits etc) serves as as a store of value (abbreviated as "sov"). Thus, a digital money token (also referred to as "sovToken") **SHOULD** be:
```

Comment:
The uppercase keywords (like "SHOULD") create requirements, and should only be used in the specification section.